plugins/elanfp/fu-elanfp-usb-device.c

```diff
@@ -0,0 +1,564 @@
+/*
+ * Copyright (C) 2021
```

Comment:
You need to list either a person or a company here

Reply:
Fixed in commit 2bed27cc885298ea58a2a46baf43a58ab44aebb9